_security/access-control/index.md

```diff
@@ -22,7 +22,7 @@ Term | Description
 Permission | An individual action, such as creating an index (e.g. `indices:admin/create`). For a complete list, see [Permissions]({{site.url}}{{site.baseurl}}/security/access-control/permissions/).
 Action group | A set of permissions. For example, the predefined `SEARCH` action group authorizes roles to use the `_search` and `_msearch` APIs.
 Role | Security roles define the scope of a permission or action group: cluster, index, document, or field. For example, a role named `delivery_analyst` might have no cluster permissions, the `READ` action group for all indexes that match the `delivery-data-*` pattern, access to all document types within those indexes, and access to all fields except `delivery_driver_name`.
-Backend role | (Optional) Arbitrary strings that you specify *or* that come from an external authentication system (for example, LDAP/Active Directory). Backend roles can help simplify the role mapping process. Rather than mapping a role to 100 individual users, you can map the role to a single backend role that all 100 users share.
+Backend role | (Optional) A backend role is a specific identifier assigned to a user or group of users by an external authentication system (for example, LDAP/Active Directory). Instead of mapping permissions to individual users, you can assign these permissions to backend roles, which can significantly streamline the role mapping process. If 100 users share a common function within the organization, they can all be assigned the same backend role. With this approach you only need to map the role to the backend role identifier, rather than to each user individually. 
```

Comment:
```suggestion
Backend role | (Optional) A backend role is a specific identifier assigned to a user or group of users by an external authentication system such as LDAP/Active Directory. Instead of mapping permissions to individual users, you can assign these permissions to backend roles, which can significantly streamline the role mapping process. For example, ff 100 users within an organization share a common function, they can all be assigned the same backend role. With this approach you only need to map the role to the backend role identifier, rather than map to each user individually. 
```